worker/worker.ts

```diff
@@ -35,7 +35,8 @@ export function startWorker() {
         }),
     );
 
-    scheduleManualJobs();
+    scheduleJobs();
+    createAlerts();
```

Comment:
we could also think about running this on CI ? but also fine here

Reply:
Yeah true. There are other things we should consider to put into a CI e.g. stellate config.